<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sproducto</class>
 <widget class="QDialog" name="sproducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bajas de Producto</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Bajas de Producto</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>204</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="productocomboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cantidad Actual:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cantidad:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="a">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="aceptar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>105</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
    <property name="icon">
     <iconset resource="pandasoft.qrc">
      <normaloff>:/images/dialog-ok.png</normaloff>:/images/dialog-ok.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>105</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="pandasoft.qrc">
      <normaloff>:/images/dialog-cancel.png</normaloff>:/images/dialog-cancel.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>productocomboBox</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>aceptar</tabstop>
  <tabstop>cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="pandasoft.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelar</sender>
   <signal>clicked()</signal>
   <receiver>sproducto</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
